process/block/metablock.go

```diff
@@ -396,6 +396,7 @@ func (mp *metaProcessor) CommitBlock(
 		return err
 	}
 
+	//TODO: Should be analyzed if this action is really necessary or not (right now there is no action of removing them)
```

Comment:
it is necessary to save into storage. should add remove from pool as well.

Reply:
Yes, in storer is necessary but I was referring here if it is necessary to put it in the pool. Changed the comment of the TODO